app/views/spree/home/index.html.erb

```diff
@@ -1,22 +0,0 @@
-<% if @homepage %>
```

Comment:
why did you remove this file. Default file should be present

Reply:
sir i removed it temporarily because i was putting static html in it i thought i will retrieve it once i will start working on index page